repos/system_upgrade/el8toel9/actors/checkipaserver/libraries/checkipaserver.py

```diff
@@ -0,0 +1,67 @@
+from leapp import reporting
+
+# FIXME: update the URL to the rhel9 document when it becomes available
+MIGRATION_GUIDE = (
+    "https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux"
+    "/8/html/installing_identity_management/migrate-7-to-8_migrating"
+)
+
+
+def ipa_inhibit_upgrade(ipainfo):
+    """
+    Create upgrade inhibitor for configured ipa-server
+    """
+    entries = [
+        reporting.Title(
+            "ipa-server does not support in-place upgrade from RHEL 8 to 9"
+        ),
+        reporting.Summary(
+            "An IdM server installation was detected on the system. IdM "
+            "does not support in-place upgrade to RHEL 9. Please follow "
+            "the migration guide lines."
+        ),
+        reporting.Remediation(
+            hint="Please follow the IdM RHEL 8 to 9 migration guide lines."
+        ),
```

Comment:
Regarding documentation rules, we should omit "please" in texts. As well, in this case, having the same sentence two times could be like Deja Vu for readers.

```suggestion
        reporting.Summary(
            "An IdM server installation was detected on the system. IdM "
            "does not support in-place upgrade to RHEL 9.
        ),
        reporting.Remediation(
            hint="Follow the IdM RHEL 8 to 9 migration guide lines."
        ),
```